testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/SFSBHibernate2LcacheStats.java

```diff
@@ -79,6 +79,7 @@ public void setupConfig() {
             Configuration configuration = new Configuration().setProperty(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS,
                     "true");
             configuration.getProperties().put(AvailableSettings.JTA_PLATFORM, JBossAppServerJtaPlatform.class);
+            configuration.getProperties().put(AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY, "jta");
```

Comment:
https://paste.fedoraproject.org/491994/03435341 contains the failure that we get without the AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY setting.  I need to find the related emails where we discussed this, as I don't want to go memory. :)